torch/distributions/__init__.py

```diff
@@ -58,7 +58,7 @@
 reparameterization trick from the
 :meth:`~torch.distributions.Distribution.rsample` method, where the
 parameterized random variable can be constructed via a parameterized
-deterministic function of a parameter-free random variable. The reparameterized
+deterministic function and a parameter-free random variable. The reparameterized
```

Comment:
I think this is incorrect, and that the existing doc string is correct.

Reply:
a parameterized deterministic function of a parameter-free random variable?
I think it should be a parameterized deterministic function and a parameter-free random variable,
the parameterized random variable can be obtained by a parameter-free random variable and a deterministic function

Reply:
Oh I see, the confusion. I think the intention was to say:
```via a parameterized deterministic transformation of a parameter-free random variable```